internal/report/report.go

```diff
@@ -428,7 +432,13 @@ func PrintAssembly(w io.Writer, rpt *Report, obj plugin.ObjTool, maxFuncs int) e
 
 		ns := annotateAssembly(insts, sns, s.base)
 
-		fmt.Fprintf(w, "ROUTINE ======================== %s\n", s.sym.Name[0])
+		if strings.HasPrefix(s.sym.Name[0], "__Z") {
+			// Workaround to handle double leading underscores on Mac
+			// which are not properly handled by the demangle.Filter
```

Comment:
Could we open an issue against the demangle package for this?

Reply:
https://github.com/ianlancetaylor/demangle/issues/12